tutorials/choropleth/tutorial.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Choropleth",
+  "hideNavbar": true,
+  "level": 0,
+  "order": 3,
+  "tutorialCss": [],
+  "tutorialJs": [],
+  "about": {
+    "text": "Create a choropleth map"
```

Comment:
This is the text shown when hovering over a tutorial, so I'd like it to be a bit more verbose.  Perhaps `Create a choropleth map, set the colors used, and add a legend.`